ocs/cluster.py

```diff
@@ -0,0 +1,347 @@
+"""
+A module for all rook functionalities and abstractions.
+
+This module has rook related classes, support for functionalities to work with
+rook cluster. This works with assumptions that an OCP cluster is already
+functional and proper configurations are made for interaction.
+"""
+
+import logging
+import base64
+
+import resources.pod as pod
+from resources import ocs
+import ocs.defaults as default
+from utility.utils import TimeoutSampler
+from ocsci import config
+from ocs import ocp
+from ocs import exceptions
+
+POD = ocp.OCP(kind='Pod', namespace=config.ENV_DATA['cluster_namespace'])
```

Comment:
This can get problematic to approach config.ENV_DATA under module level with new config change provided by #210 . AFAIU if the module gets imported by python before proper load of config we won't have the clusetr_namespace overwritten by custom config, can you please confirm @zmc ? I think that if we won't import this module from any module which takes care of config or is imported by them we are OK.

Reply:
Yeah, statements like these are evaluated at import time, which is most likely before the configuration has been read.

Reply:
@shylesh might be better to move under __init__ and outside import evaluation.

Reply:
Done, moved it inside __init__